libclc/libspirv/lib/generic/math/copysign.cl

```diff
@@ -25,7 +25,7 @@ _CLC_DEFINE_BINARY_BUILTIN(double, __spirv_ocl_copysign, __builtin_copysign,
 
 #pragma OPENCL EXTENSION cl_khr_fp16 : enable
 
-_CLC_DEFINE_BINARY_BUILTIN(half, __spirv_ocl_copysign, __builtin_copysign, half,
+_CLC_DEFINE_BINARY_BUILTIN(half, __spirv_ocl_copysign, __builtin_copysignf16, half,
```

Comment:
Bear in mind that there's already [`__clc_copysign`](https://github.com/llvm/llvm-project/blob/main/libclc/clc/lib/generic/math/clc_copysign.cl) upstream, which already uses `__builtin_elementwise_copysign`. So if libspirv was properly keeping up with the upstream changes and having `__spirv_ocl_copysign` call `__clc_copysign`, we wouldn't need this change.

Reply:
Dropped this change.